ui/component/app/view.jsx

```diff
@@ -120,14 +121,14 @@ function App(props: Props) {
   const rawReferrerParam = urlParams.get('r');
   const sanitizedReferrerParam = rawReferrerParam && rawReferrerParam.replace(':', '#');
   const wrapperElement = appRef.current;
+  const shouldHideNag = pathname.startsWith(`/$/${PAGES.EMBED}`) || pathname.startsWith(`/$/${PAGES.AUTH_VERIFY}`);
```

Comment:
yuck but okay

ideally this would be handled either inside of the top-level component the route points at or concurrently with route definitions

Reply:
Do you mean something in the `EmbedPage`/`AuthVerifyPage` that hides nags?